.github/CODEOWNERS

```diff
@@ -100,6 +100,7 @@
 /neoload/                                @Neotys-Labs/r-d @DataDog/ecosystems-review
 /neutrona/                               @DavidFlamini david@neutrona.com
 /nextcloud/                              @eplanet
+/ngrok/                                  @mkarnowski
```

Comment:
Please add @DataDog/ecosystems-review to the code owners file

Reply:
requested change committed